README.md

```diff
@@ -57,81 +56,114 @@ cd ../server && npm install
+You can find the steps at the complete [State Channel Tutorial Guide](TUTORIAL.md).
 
 
-## Is State Channel Demo probably fair?
+## Is State Channel Demo provably fair?
 
 State Channels offer an innovative solution, where off-chain transactions are executed on top of blockchain technology. 
 
-State channels protocol inherently offers security to the demo.
+The State Channel protocol inherently offers security to the demo!
+
+State Channel transactions are ruled by co-signed Smart Contracts. Each party shall verify the content of a transaction before signing it. Those transactions can be off-chain such as `ContractCallTx`, `SpendTx` or on-chain such as `MutualCloseChannelTx`. All transactions need to be co-signed by both parties.
 
-State Channel Transactions are ruled by co-signed smart contracts. Each participant shall verify the content of a transaction before signing it. Those transactions can be off-chain contract calls, spend transactions or on-chain such as `MutualCloseChannelTx`. All transactions need to be co-signed by both parties.
+State Channel Demo Game verifies also counterparty transactions. At each transaction, calldata content is decoded utilizing æternity's [calldata lib](https://github.com/aeternity/aepp-calldata-js) which is integrated in the SDK. Each party can utilize this library in order to verify that the opponent is following the anticipated flow of the game (calling the right contract methods). If they accept opponent's transaction, then they can co-sign it, otherwise they will refuse to co-sign.
 
-For each participant, the provided æternity node spawns a state channel process which is responsible to validate state channel transactions through root hash. It builds and enriches State Channel Trees, tracks any possible disputes and warns each client so one could take an appropriate action.
+## How do State Channel Demo Game transactions take place?
 
-State Channel Demo game verifies also counterpart participant transactions. At each transaction callData content is decoded utlizing æternity's  [calldata lib](https://github.com/aeternity/aepp-calldata-js) which is integrated in the SDK. Participants can utilize this library in order to verify that the opponent is following the anticipated flow of the game (calling the right contract methods). If they accept opponent's transaction, then they can co-sign it, otherwise they can raise a dispute (demo game implementation showcases the happy path).
+As rock, paper, scissors is a turn based game and the move of each player needs to be co-signed by the opponent in order to be executed, the move of the initial player is hashed. The other party receives the off-chain transaction with calldata containing the hash, where he/she can **only** confirm the method the opponent called (e.g. opponent picked a move) but not see the actual move.
```

Comment:
```suggestion
Rock, paper, scissors expects the players to make their moves simultaneously, but in the State Channels implementation, it needs to be implemented as a turn based game, where the move of each player needs to be co-signed by the opponent in order to be executed. In order to prevent cheating, the move of the initial player is hashed. The other party receives the off-chain transaction with calldata containing the hash, where he/she can **only** confirm the method the opponent called (e.g. opponent picked a move) but not see the actual move.
```